flink-runtime/src/main/java/org/apache/flink/runtime/clusterframework/standalone/StandaloneMode.java

```diff
@@ -16,24 +16,21 @@
  * limitations under the License.
  */
 
-package org.apache.flink.runtime.execution;
+package org.apache.flink.runtime.clusterframework.standalone;
 
-public interface ExecutionObserver {
+/**
+ * The startup mode for the standalone setup,
+ */
+public enum StandaloneMode {
```

Comment:
You're looking at an old commit. I already removed this class but didn't want to force-push.